themes/gatsby-theme-deriv/src/pages/signup-affiliates/index.tsx

```diff
@@ -48,6 +49,8 @@ const StyledContainer = styled(Container)`
 `
 
 const AffiliateSignup = ({ pageContext }: TGatsbyHead) => {
+    const aff_token_cookie = Cookies.getJSON("referrer_token")
+    const aff_token_url = affiliateTokenURL()
```

Comment:
[M] This check is redundant, since cookie.js already handled `t` url and store it to the cookie

Reply:
yes, base on requirement get it from cookies then check url, @habib-deriv pls let me know if no need to check url anymore 🙏